docker-compose.yml

```diff
@@ -27,11 +25,7 @@ services:
                         APP_NAME: HMP Berwyn
```

Comment:
Is it worth changing the default app name to something like "HMP Default".  That way we can easily pick up overrides that haven't been set properly?

Reply:
Yeah that could work well actually, will make that change